ansible/roles/common-packages/defaults/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+common_packages:
+  - { name: acl, version: "2.3.2-1build1.1" }
```

Comment:
We are using two different variable structures to define a list of `package:version`:
```
- { name: autoconf, version: "2.71-3" }
```
and [here](https://github.com/nebius/soperator/pull/1742/files#diff-f78510cd5ab767f25ddda175050b6a866b081471b0cfd8cb44069f04b063cd17R6):
```
- "cuda=12.9.0-1"
```

It would be better to converge into one.